### PR TITLE
Fix a lot of french sentences

### PR DESCRIPTION
**Pull request by Booteille**
Fix a lot of spelling mistakes and removed a ninja sentence duplication.

### CODE REVIEW

**Comment by mikehenrty** [comment]
@lissyx could you take a look?

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 1]
```diff
@@ -901,7 +901,7 @@ dans ton bureau.
 dans tes mains.
 dans ta maison.
 dans ta main.
-nous avons besoin de bénévoles prêts à nous aider ou à nous proposer de nouvelles idées pour organiser des évènements de qualité.
+nous avons besoin de bénévoles prêts à nous aider ou à nous proposer de nouvelles idées pour organiser des événements de qualité.
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: évènements est également valide

File: server/data/fr/ofis_publik_ar_brezhoneg.txt [hunk 2]
```diff
@@ -1104,7 +1104,7 @@ elle, elle lirait.
 elle, elle lira.
 elle, elle a été aimée.
 elle, elle est aimée.
-elle a l'esprit affuté.
+elle a l'esprit affûté.
```
Comment (lissyx) [inline server/data/fr/ofis_publik_ar_brezhoneg.txt]: Both are valids

**Comment by Booteille** [comment]
Oh, that looks like something Libreoffice did alone, sorry.

**Comment by zecakeh** [comment]
While we are at it we should fix the missing capitalized first letter of the sentences.
Also there are a lot of duplicates (and I don't mean the sentences where only one word is changed, real duplicates). They are really hard to miss since they are usually grouped together.

**Comment by lissyx** [comment]
@zecakeh I'm not sure it's really a big deal for the lack of capitalization. Duplicates should be easy to find with `sort | uniq` call

**Comment by Booteille** [comment]
I wanted to concentrate on the first draw only on wrong typos but if someone have time to apply another fix, feel free to do it! For my part, i'll not have the time for next days.

**Comment by lissyx** [comment]
I'm also busy right now, I might also be able to have a look at that in the next days though.

**Comment by drzraf** [comment]
mentioning #1191

**Comment by zecakeh** [comment]
@lissyx I'm sure it's not a big deal for the AI, however it feels nicer when you're recording/validating (for me anyway).
I'm looking at the file and am planning to be done by tomorrow or monday.

**Comment by zecakeh** [comment]
According to #1191 (if I got it right), we're not supposed to remove duplicates because they want to keep the same sentences between the french version and the breton version. That's a major setback for me since a the first part of my work was removing those duplicates.
So I won't do anything until I know what is supposed to be corrected.

**Comment by lissyx** [comment]
@zecakeh Maybe, but then the original codebase should be fixed instead of just this output file, otherwise the next import might overwrite that.

**Comment by mikehenrty** [comment]
> @mikehenrty I think I read somewhere that we are supposed to spell the numbers instead of keeping the numeric form. Am I wrong ? Also I guess there's no point in keeping the abbreviations like n° for "numéro" or cf for "confer".

Exactly right, spelling out the numbers is preferred, and best to expand all abbreviations.

**Comment by mikehenrty** [comment]
> According to #1191 (if I got it right), we're not supposed to remove duplicates because they want to keep the same sentences between the french version and the breton version. That's a major setback for me since a the first part of my work was removing those duplicates.

Thanks for flagging this. We have decided that you can remove duplicates. See my comment for more info: https://github.com/mozilla/voice-web/pull/1191#issuecomment-401748267

**Comment by lissyx** [comment]
@zecakeh About spelling numbers and abbreviations, this is stuff I already dealt with in https://github.com/lissyx/commonvoice-syceron/tree/master/, any contirbution is welcome to make that a more generic, more robust tooling.

Anyone interested in helping contribution on french should check https://github.com/mozfr/besogne/wiki/Common-Voice-Fr

**Comment by zecakeh** [comment]
Added a PR on @Booteille 's fork with all the fixes I mentioned before : [Fix and cleanup all sentences](https://github.com/Booteille/voice-web/pull/1).
I took @lissyx 's good advice to keep valid but unusual spellings.
@drzraf 's modifications in PR #1191 are also included in my commit.

From what I see we're gonna have conflicts with some merged PRs.

@lissyx Good to know. It's hard to know where all the informations are since they are not centralized. Also I might have a look at your tool one of these days, looks promising!

**Comment by lissyx** [comment]
@zecakeh Please note that Breton's originated strings are actually from @ftyers, and not my work :-). Can someone send an updated / rebased PR ?

**Comment by mikehenrty** [comment]
fixed in #1254

**Pull request closed without merging**